src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Square extends Figure implements Shape {
+    private int sideA;
```

Comment:
what is A?

Reply:
it is the size of the sides of the square. I will cgange the name of variable